package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@commerce7/admin-ui",
-  "version": "2.0.4",
+  "version": "2.0.45",
```

shound't this be 
```suggestion
  "version": "2.0.5",
```
instead?